reactioncommerce/reaction
# feat: update to Meteor 1.7

Resolves #4239   
Impact: **minor**  
Type: **feature**

Updated to Meteor 1.7

## Breaking changes
Probably none, but if you've added extra Babel packages, be sure to update them to the same version as all the included Babel packages.

## Testing
1. Make sure there are no errors on startup when using `reaction` command to run.
1. Make sure there are no errors on startup when running with docker-compose.
1. Run `npm test`. Make sure no errors appear on client or server and all tests pass.
